guide/src/componentDocs/Identity/IdentityDoc.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { P } from 'cyverse-ui';
+import meta from 'cyverse-ui/Identity.json';
+import { Figure, ComponentDoc, CodeBlock } from '../../components';
```

Comment:
Same here - I think `... from 'components'` works because `guide/src` is the defined [`context` in webpack](https://github.com/cyverse/cyverse-ui/blob/master/guide/webpack.config.js#L12) for the guide.

I only mention it since this is new code (or appears to be).